tests/display/src/XmlDisplayTestCase.hx

```diff
@@ -10,6 +10,12 @@ class XmlDisplayTestCase implements utest.ITest {
 
 	public function new() {}
 
+	public function setup() {
+		var methodArgs = {method: haxe.display.Protocol.Methods.ResetCache, id: 1, params: {}};
```

Comment:
Minor thing but this looks like a prime opportunity for a local `static var`.

Reply:
Sounds like a good excuse to trigger another CI test run, too